Source/WebInspectorUI/UserInterface/Views/GradientSlider.js

```diff
@@ -416,6 +406,7 @@ WI.GradientSliderKnob = class GradientSliderKnob extends WI.Object
 
         if (this._detaching)
             this.y = event.pageY - this._startMouseY;
+            this._updateTransform();
         else if (this.delegate && typeof this.delegate.knobXDidChange === "function")
```

Comment:
you need to wrap this with `{` `}` or it's not valid JS since you have more than one line inside the body of the `if`

also, you need to change it to set `this._y` since that's what's used by `WI.GradientSliderKnob.prototype._updateTransform`